app/models/miq_worker/runner.rb

```diff
@@ -374,6 +375,32 @@ def heartbeat_to_drb
   def heartbeat_to_file(timeout = nil)
     timeout ||= worker_settings[:heartbeat_timeout] || Workers::MiqDefaults.heartbeat_timeout
     File.write(@worker.heartbeat_file, (Time.now.utc + timeout).to_s)
+
+    get_messages.each { |msg, *args| process_message(msg, *args) }
+  end
+
+  def get_messages
+    messages = []
+    now = Time.now.utc
+    @my_last_config_change ||= now
```

Comment:
Can we simplify these to lines to this?

```ruby
@my_last_config_change ||= Time.now.utc
```

That way, we are only calling `Time.now.utc` if `@my_last_config_change` isn't set, and it doesn't look like the `now` variable is being used elsewhere.